README.md

```diff
@@ -147,10 +153,31 @@ part of `users` will be authorized to access the related index.
 
 ### Start docker-compose
 
+Start Monocle:
+
 ```ShellSession
 $ docker-compose up -d
 ```
 
+Ensure services are running:
+
+```ShellSession
+$ docker-compose ps
+monocle_api_1       uwsgi --uid guest --gid no ...   Up      0.0.0.0:9876->9876/tcp, 0.0.0.0:9877->9877/tcp
+monocle_crawler_1   monocle --elastic-conn ela ...   Up
+monocle_elastic_1   /usr/local/bin/docker-entr ...   Up      0.0.0.0:9200->9200/tcp, 9300/tcp
+monocle_web_1       docker-entrypoint.sh /bin/ ...   Up      0.0.0.0:3000->3000/tcp
+```
+
+After a change in the configuration file, the api and crawler services need to be restarted:
```

Comment:
isn't it automatic in docker-compose up -d ?

Reply:
I just tested and no it doesn't.

Reply:
ok merging it.